t/array-index-write.neon

```diff
@@ -0,0 +1,7 @@
+VAR a: Array<Number> := []
+VAR x: Number := 0
+a[5] := 6
+x := a[5]
+
+print("\(a.size()), \(x)")
+%= 6, 6
```

Comment:
Please make sure you don't commit files with missing newlines at the end.

Reply:
Opps.  That was a mistake.  Fixed.